Antidote/AbstractCallViewController.h

```diff
@@ -34,6 +36,8 @@
  */
 @property (weak, nonatomic) OCTSubmanagerCalls *manager;
 
+@property (weak, nonatomic) AppearanceManager *appearanceManager;
```

Comment:
I was trying to avoid singletons, but then I just realized `Appcontext` is imported everywhere, so I guess I should just use it then.